### DISCUSSION TITLE
Update Home Assistant Entity Ids after renaming in Raspberrymatic

### DISCUSSION THREAD

**Discussion by SimonCW** [OpenCCU/OpenCCU#1999] [🇺🇸 Q&A] [answered]
Hey,

can I somehow update the Entity-IDs in HomeAssistant from the Group names in Raspberrymatic? 

Long:
I renamed my Groups in Raspberrymatic, e.g. "hg_kitchen" for *heating group kitchen*. Now, I also want the Entity-Id in Home Assistant to be "hg_kitchen" so that I can easily use it in templates, etc. However, in Home Assistant the Entity-Ids are still the first synced name, e.g. "climate.hmip_heating_int0000003". 
Do I have to change entity Ids manually on the Integrations Page or can I sync this automatically?

**Comment by SimonCW** [comment, marked as answer]
Edit: 
As mentioned in the comments by @SukramJ, this is the way to go: https://github.com/danielperna84/hahomematic/wiki/Howto's

Original
Solved it by removing the Homematic IP (local) integration and re-adding it. 

Warning: You'll have to update the Entity-IDs in your dashboards, etc.

**Reply by SukramJ** [reply]
Sorry this is not a good solutions. Once you'll use device triggers or actions you are lost.
Option 3 in this [Howto](https://github.com/danielperna84/hahomematic/wiki/Howto's) should be more appropriate.

**Reply by SimonCW** [reply]
@SukramJ, your link is referring to an integration I don't have. Can you explain how it relates to https://github.com/danielperna84/custom_homematic?

**Reply by SukramJ** [reply]
This is the backend lib used by custom_homematic. It is mentioned in the docs.
